ios-trivia-game/ViewControllers/QuestionViewController.swift

```diff
@@ -18,12 +18,19 @@ class QuestionViewController: UIViewController {
         // Do any additional setup after loading the view.
         
         // @Nari or @Zhia: Feel free to remove the hardcoded values when you get to this
-        let questionId = 1932
+        let questionId = 1938
```

Comment:
this should be removed once questionviewcontroller PR is checked in.  For now, I guess it is okay?